docs/en/drivers/windows.md

```diff
@@ -0,0 +1,136 @@
+3. Once created, select **Project > Manage NuGet Packages... > Browse** and
+   search for **Appium.WebDriver**
+4. Install the **Appium.WebDriver** NuGet packages for the test project
+5. Start writing your test (see sample code under [samples])
+
+#### Universal Windows Platform App Testing
+
+To test a UWP app, you can use any Selenium supported language and simply
+specify the **Application Id** for the app under test in the **app**
+capabilities entry. Below is an example of creating a test session for Windows
+**Alarms & Clock** app written in C#:
+
+```c#
+// Launch the AlarmClock app
+DesiredCapabilities appCapabilities = new DesiredCapabilities();
+appCapabilities.SetCapability("app", "Microsoft.WindowsAlarms_8wekyb3d8bbwe!App");
+AlarmClockSession = new WindowsDriver<WindowsElement>(new Uri("http://127.0.0.1:4723"), appCapabilities);
+// Control the AlarmClock app
+AlarmClockSession.FindElementByAccessibilityId("AddAlarmButton").Click();
+AlarmClockSession.FindElementByAccessibilityId("AlarmNameTextBox").Clear();
+```
+
+When testing the application you authored yourself, you can find the **Application Id** in the generetated `AppX\vs.appxrecipe` file under `RegisteredUserNmodeAppID` node. E.g. ```c24c8163-548e-4b84-a466-530178fc0580_scyf5npe3hv32!App```
+
+#### Classic Windows App Testing
+
+To test a classic Windows app, you can also use any Selenium supported language
+and specify the **full executable path** for the app under test in the **app**
+capabilities entry. Below is an example of creating a test session for Windows
+**Notepad** app:
+
+```c#
+// Launch Notepad
+DesiredCapabilities appCapabilities = new DesiredCapabilities();
+appCapabilities.SetCapability("app", @"C:\Windows\System32\notepad.exe");
+NotepadSession = new WindowsDriver<WindowsElement>(new Uri("http://127.0.0.1:4723"), appCapabilities);
+// Control the AlarmClock app
+NotepadSession.FindElementByClassName("Edit").SendKeys("This is some text");
+```
+
+#### Starting a Session
+
+As mentioned above, you should additionally use these capabilities to ensure
+you are getting a Windows App automation session:
+
+`platformName`: `Windows`
+`deviceName`: `WindowsPC`
+`app`: the appID of the Windows app for testing, or the path to the .exe file
+
+#### Inspecting UI Elements
+
+Microsoft Visual Studio 2015 by default includes Windows SDK that provides
+great tool to inspect the application you are testing. This tool allows you to
+see every UI element/node that you can query using Windows Application Driver.
+This **inspect.exe** tool can be found under the Windows SDK folder such as
+`C:\Program Files (x86)\Windows Kits\10\bin\x86`. The tool will show various
+element attributes. The table below shows you witch Appium locator strategy you
+should use to find elements with the corresponding attributes.
+
+| Locator Strategy| Matched Attribute|
+|-----------------|------------------|
+| accessibility id|   AutomationId   |
+|    class name   |     ClassName    |
```

Comment:
To center this automatically, you can make this the second row:

|:-----------------:|:-----------------:|

Reply:
good thought. i wasn't trying to center it, just fix what was there before.